src/Illuminate/Session/Store.php

```diff
@@ -32,6 +32,13 @@ class Store implements Session
      */
     protected $attributes = [];
 
+    /**
+     * The session attributes when started.
+     *
+     * @var array
+     */
+    protected $initAttributes = [];
```

Comment:
In light of clarity, I suggest `$original` which would follow the same convention as established in `\Illuminate\Database\Eloquent\Concerns\HasAttributes`.

Or, if too vague in this context, `$originalAttributes`.

Or if not favored: `$initialAttributes`

Somehow my brain does not like `$init…` 🤷‍♀️